exporter/otlp-http/Gemfile

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source 'https://rubygems.org'
+
+gemspec
+
+# Use the opentelemetry-api gem from source
+gem 'opentelemetry-api', path: '../../api'
+gem 'opentelemetry-common', path: '../../common'
+gem 'opentelemetry-exporter-otlp-common', path: '../otlp-common'
+gem 'opentelemetry-instrumentation-base', path: '../../instrumentation/base'
```

Comment:
Is this necessary for this gem? Is `opentelemetry-common` needed?

Reply:
They get pulled in implicitly from the SDK regardless.  But no we should not need them explicitly here with the decoupling changes being made for extracting instrumentation in the contrib repo.